README.txt

```diff
@@ -0,0 +1,24 @@
+# WarpX Data
+
+## Overview
+
+This repository contains data files that are too large for the main [WarpX source directory](https://github.com/ECP-WarpX/WarpX).
+
+## Documentation
+
+This section describes the directory structure:
+
+...
```

Comment:
@roelof-groenewald can you complete this section and make this again a `.md` file? :)